models/LogLevel.js

```diff
@@ -1,11 +1,10 @@
 module.exports = require('./Enum').create({
   values: [
-    'FATAL',
-    'ERROR',
-    'WARN',
-    'INFO',
-    'DEBUG',
-    'TRACE'
-  ],
-  autoLowerCase: true
```

Comment:
Hmm, okay. Looks like the behavior isn't consistent for lowercase vs uppercase coercion and that caused confusion. https://github.com/fashion-js/fashion-model/blob/master/Enum.js#L119 will cause lowercased enum values to be available via upper or lowercased properties. Since I was using lowercase in my setup, I assumed the option was broken and didn't do anything. Assuming lowercase, nothing prevents one from specifying upper or lowercase when instantiating `Config`, either works just fine and the resulting value will automatically get coerced to lowercase. E.g.


**LogLevel.js**
```js
module.exports = require('./Enum').create({
  values: ['fatal', ...] // All of these are lowercase
})
```

**Config.js**
```js
module.exports = require('./Model').extend({
  properties: {
    logLevel: LogLevel,
    ...
  }
})
```

**Creating the config**
```js
const config = new Config({
  logLevel: 'FATAL' // Automagically gets converted to lowercase and works (all lowercase works too)
})
```